### ISSUE TITLE
[a11y] Incomplete DaisyUI color settings

### ISSUE THREAD

**Issue by EvgeniiaVak** [DeXter-on-Radix/website#245]
@EvgeniiaVak adapted to tailwind like styles.

Concerning changing the style of the outline, we should be careful to keep a high contrast, as it otherwise would be contraproductive to accessibility:
<img width="703" alt="accessibility-low-vision" src="https://github.com/DeXter-on-Radix/website/assets/44790691/fbb735e0-610f-4bf4-a274-4c74c57b1caf">
(source: section [accessibility_concerns](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#accessibility_concerns) from official documentation)

If we want to change the styling to brand colors I'd suggest to track that in a separate issue/PR as this would involve changing all `focus-visible` borders on the page to keep a consistent style overall.

Maybe an alternative solution is to find out why pressing `esc` triggers the `focus-visible` state to be activated in the first place and prevent that using JS, WDYT? 🤔 

And thanks a lot for the comment, I realized how inconvenient using the main branch is after trying to submit a second PR! 🤣 (I guess thats a reflex from using mercurial in the past, where you barely work with branches and always commit to main)

_Originally posted by @dcts in https://github.com/DeXter-on-Radix/website/issues/242#issuecomment-1922355223_
